src/components/renderer/index.css

```diff
@@ -32,4 +32,52 @@
 .chart .vega-bind-radio label {
   margin-left: 0;
   margin-right: 4px;
+}
+
```

Comment:
You mean something like this `../../../node_modules/vega-tooltip/build/vega-tooltip.css`?